docs/guide/start-installation.md

```diff
@@ -5,6 +5,53 @@ Installation
 
 The minimum requirement by this project template is that your Web server supports PHP 5.4.0.
 
+## Installing using Vagrant
+
+This way is the easiest but long (~20 min).
+
+**This installation way doesn't require pre-installed software (such as web-server, PHP, MySQL etc.)** - just do next steps!
+
+1. Install [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+2. Install [Vagrant](https://www.vagrantup.com/downloads.html)
+3. Windows users must reboot now
+4. Create GitHub [personal API token](https://github.com/blog/1509-personal-api-tokens)
+5. Prepare project
+   1. Linux/Unix users can run commands:
+   
+      ```bash
+      git clone https://github.com/yiisoft/yii2-app-advanced.git
+      cd yii2-app-advanced/vagrant/config
+      cp vagrant-local.example.yml vagrant-local.yml
+      ```
+   2. Windows users:
+      * download repo [yii2-app-advanced](https://github.com/yiisoft/yii2-app-advanced/archive/master.zip)
+      * unzip it
+      * go into directory `yii2-app-advanced-master/vagrant/config`
+      * copy `vagrant-local.example.yml` to `vagrant-local.yml`
+   
+6. Place your GitHub personal API token to `vagrant-local.yml`
+7. Windows users must manually add the following lines to [hosts file](https://ru.wikipedia.org/wiki/Hosts):
```

Comment:
Should use `https://en.wikipedia.org/wiki/Host` with English instead `https://ru.wikipedia.org/wiki/Hosts` ?